templates/docs/patterns/cta/block/index.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Not sure why this ended up documented as `/cta/block` - it should use full component name in the path without spliting it, so `/cta-block`